build.gradle

```diff
@@ -20,11 +20,14 @@ repositories {
 }
 
 dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
-	developmentOnly 'org.springframework.boot:spring-boot-devtools'
-	testImplementation('org.springframework.boot:spring-boot-starter-test') {
-		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
-	}
+	implementation 'org.springframework.boot:spring-boot-starter-security'
+	compile("com.auth0:java-jwt:3.4.0")
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation 'org.springframework.security:spring-security-test'
+//	compile group: 'org.springframework.security', name: 'spring-security-core', version: '5.2.2.RELEASE'
```

Comment:
Commented dependencies should probably be removed.

Reply:
Done